### PR TITLE
LF: cleanup snapshot replay + add minimal doc

### PR DESCRIPTION
**Pull request by remyhaemmerle-da**
CHANGELOG_BEGIN
CHANGELOG_END

### Pull Request Checklist

- [ ] Read and understand the [contribution guidelines](https://github.com/digital-asset/daml/blob/main/CONTRIBUTING.md)
- [ ] Include appropriate tests
- [ ] Set a descriptive title and thorough description
- [ ] Add a reference to the [issue this PR will solve](https://github.com/digital-asset/daml/issues), if appropriate
- [ ] Include changelog additions in one or more commit message bodies between the `CHANGELOG_BEGIN` and `CHANGELOG_END` tags
- [ ] Normal production system change, include purpose of change in description
- [ ] If you mean to change the status of a component, please make sure you keep [the Component Status page](https://github.com/digital-asset/daml/blob/main/docs/source/support/component-statuses.rst) up to date.

NOTE: CI is not automatically run on non-members pull-requests for security
reasons. The reviewer will have to comment with `/AzurePipelines run` to
trigger the build.

### CODE REVIEW

File: daml-lf/snapshot/README.md [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+      -p choiceName=<exercise choice names>             \
+      [-p choiceIndex=<index of the exercise node>]     \
+      [-p darFile=<dar files>]                                              
+
+
+where:
+
+* `<entries files>`: is the full path of the ledger entries
+  files to be used separated by commas (`,`)
+
```
Comment (nickchapman-da) [inline daml-lf/snapshot/README.md:27]: How does one get hold of an _entries file_ ?

File: daml-lf/snapshot/README.md [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+  program. The adaptation process attempts to map the identifiers
+  from the export file with the ones of dar file when those latter
+  differ only in their package ID.  This can be used when the original
+  Daml source used to generate the ledger export is only slightly
+  modified or compiled with different options.
+
+
+# `replay-profile`
+
+This profile the replay od a transaction from a ledger entries stored in a file.
```
Comment (nickchapman-da) [inline daml-lf/snapshot/README.md:53]: text is a bit garbled here

**Review by nickchapman-da — APPROVED** [review]
LGTM
apart from a small typo

File: daml-lf/snapshot/README.md [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+  program. The adaptation process attempts to map the identifiers
+  from the export file with the ones of dar file when those latter
+  differ only in their package ID.  This can be used when the original
+  Daml source used to generate the ledger export is only slightly
+  modified or compiled with different options.
+
+
+# `replay-profile`
+
+This profile the replay of a transaction built from a ledger entries file.
```
Comment (nickchapman-da) [inline daml-lf/snapshot/README.md:55]: "This profiles ..."

Comment (remyhaemmerle-da) [inline daml-lf/snapshot/README.md:55]:
```suggestion
This profiles the replay of a transaction built from a ledger entries file.
```

**Review by sofiafaro-da — APPROVED** [review]
Thanks!

**Pull request merged**